### ISSUE TITLE
Colour change menu crashes game

### ISSUE THREAD

**Issue by Revenent666** [ate47/Xray#73]
### Mod version

2.0.5

### Minecraft version

1.19.2

### Mod loader

Fabric

### Description of the issue

Whenever I try to change the colour and enter the colour changing selection screen. It crashes

### Excepted behavior

to be able to change the colour

### How to reproduce

- Enter ESP Config
- Click on name of the titles for the ESP selections
- Click on colour change

**Comment by ate47** [comment]
True, I might fix that later

When we hover the buttons

```java
Reported exception thrown!
net.minecraft.class_148: Rendering screen
	at net.minecraft.class_757.method_3192(class_757.java:890)
	at net.minecraft.class_310.method_1523(class_310.java:1176)
	at net.minecraft.class_310.method_1514(class_310.java:767)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.IllegalStateException: Not building!
	at net.minecraft.class_287.method_43577(class_287.java:243)
	at net.minecraft.class_287.method_1326(class_287.java:235)
	at fr.atesab.xray.utils.GuiUtils.drawGradientRect(GuiUtils.java:574)
	at fr.atesab.xray.utils.GuiUtils.drawGradientRect(GuiUtils.java:517)
	at fr.atesab.xray.utils.GuiUtils.drawBox(GuiUtils.java:488)
	at fr.atesab.xray.utils.GuiUtils.drawTextBox(GuiUtils.java:466)
	at fr.atesab.xray.screen.ColorSelector.lambda$render$4(ColorSelector.java:261)
	at fr.atesab.xray.screen.ColorSelector.method_25394(ColorSelector.java:286)
	at net.minecraft.class_757.method_3192(class_757.java:881)
```

**Comment by ate47** [comment]
Fixed in Fabric 1.19 2.1.0, need Forge port
